vue-components/tests/unit/components.spec.ts

```diff
@@ -0,0 +1,13 @@
+import { mount } from '@vue/test-utils';
+import * as allComponents from '@/main';
+
+describe( 'All components', () => {
+	it( 'have the class wikit in their root node', () => {
+		Object.keys( allComponents ).forEach( ( component ) => {
```

Comment:
```suggestion
		Object.values( allComponents ).forEach( ( component: VueConstructor<Vue> ) => {
```
(with `import Vue, { VueConstructor } from 'vue';`)

would be slightly nicer in my eyes, but it's no big deal.

It's not clear to me why `Object.values()` is frowned upon though (es/no-object-values).